frontend/src/components/login/Login.tsx

```diff
@@ -0,0 +1,235 @@
+          <Input
+            isClearable
+            isRequired
+            fullWidth
+            onInput={(e) => {
+              setEmail(e.currentTarget.value);
+            }}
+            placeholder="Email"
+          />
+          <Spacer y={1} />
+          <Input
+            type={isPasswordVisible ? "text" : "password"}
+            placeholder="Password"
+            isClearable
+            isRequired
+            fullWidth
+            onInput={(e) => {
+              setPassword(e.currentTarget.value);
+            }}
+            endContent={
+              isPasswordVisible ? (
+                <Button
+                  variant="light"
+                  className="focus:outline-none"
+                  size="sm"
+                  isIconOnly
+                  onClick={togglePasswordVisibility}
+                >
+                  <EyeHideLogo height="50%" />
+                </Button>
+              ) : (
+                <Button
+                  variant="light"
+                  className="px-2 focus:outline-none"
+                  size="sm"
+                  isIconOnly
+                  onClick={togglePasswordVisibility}
+                >
+                  <EyeShowLogo height="50%" />
+                </Button>
+              )
+            }
+          />
+          {isSignUp ? (
+            <div className="items-center">
+              <Spacer y={1} />
+              <Input
+                type={isCheckPasswordVisible ? "text" : "password"}
+                placeholder="Re-enter password"
+                isClearable
+                isRequired
+                fullWidth
+                onInput={(e) => {
+                  setCheckPassword(e.currentTarget.value);
+                }}
+                endContent={
+                  isCheckPasswordVisible ? (
+                    <Button
+                      variant="light"
+                      className="focus:outline-none"
+                      size="sm"
+                      isIconOnly
+                      onClick={() => toggleCheckPasswordVisibility()}
+                    >
+                      <EyeHideLogo height="50%" />
+                    </Button>
+                  ) : (
+                    <Button
+                      variant="light"
+                      className="px-2 focus:outline-none"
+                      size="sm"
+                      isIconOnly
+                      onClick={() => toggleCheckPasswordVisibility()}
+                    >
+                      <EyeShowLogo height="50%" />
+                    </Button>
+                  )
+                }
+              />
+              <Spacer y={1} />
+              <Input
+                isClearable
+                isRequired
+                fullWidth
+                onInput={(e) => {
+                  setName(e.currentTarget.value);
+                }}
+                placeholder="Name"
+              />
+              {password != checkPassword ? (
```

Comment:
I will suggest we always use `!==` instead of `!=`. The difference is that operators like `!==` or `===` will not only check the value but also the type of the variables that you are comparing.